diff --git a/browser/chrome/browser/res/activity-stream/data/content/abouthomecache/page.html.template b/browser/chrome/browser/res/activity-stream/data/content/abouthomecache/page.html.template
index a06b7f8..6e29db6 100644
--- ./browser/chrome/browser/res/activity-stream/data/content/abouthomecache/page.html.template
+++ ./browser/chrome/browser/res/activity-stream/data/content/abouthomecache/page.html.template
@@ -25,5 +25,5 @@
     <script async type="module" src="chrome://global/content/elements/moz-toggle.mjs"></script>
     <script async type="module" src="chrome://global/content/elements/moz-button.mjs"></script>
     <script async type="module" src="chrome://global/content/elements/moz-button-group.mjs"></script>
-  <div>hello</div></body>
+  <div>hello</div><div>hello</div></body>
 </html>
